capi/VideoDecoderCapi.cpp

```diff
@@ -18,8 +18,8 @@
 #include "config.h"
 #endif
 #include "VideoDecoderCapi.h"
-#include "interface/VideoDecoderInterface.h"
-#include "interface/VideoDecoderHost.h"
+#include "VideoDecoderInterface.h"
+#include "VideoDecoderHost.h"
```

Comment:
👎 in the current form.  @uartie can you please take a look at this?  I remember discussing about this with you in the past.